keyboards/handwired/dactyl_manuform/5x6/keymaps/squirrel/keymap.c

```diff
@@ -0,0 +1,106 @@
+/* A standard layout for the Dactyl Manuform 5x6 Keyboard */
+
+#include QMK_KEYBOARD_H
+
+extern keymap_config_t keymap_config;
+
+#define _MAC 0
+#define _WINDOWS 1
+#define _MOUSE 2
+#define _ARROWS 3
+#define _SYMBOLS 4
+#define _NUMBERS 5
+#define _MEDIA 6
```

Comment:
We recommend using an `enum` to manage the layer indexes, as it's less labor intensive. Lines 7-13 should be replaced with:

```c
enum layer_names {
    _MAC,
    _WINDOWS,
    _MOUSE,
    _ARROWS,
    _SYMBOLS,
    _NUMBERS,
    _MEDIA,
};
```

This will compile to the same result as your #define stanza, but removes the requirement of renumbering the layers manually if you change your layers at all.